ground/gcs/src/plugins/picoc/picocgadgetwidget.cpp

```diff
@@ -385,6 +385,17 @@ void PicoCGadgetWidget::sleep(int msec)
 }
 
 /**
+* @brief PicoCGadgetWidget::on_tbTestValueSend_clicked  Send the Test Value
+ */
+void PicoCGadgetWidget::on_tbTestValueSend_clicked()

```

Comment:
Thanks for your feedback,
i also had to search a little bit how that auto connect for "Signal/Slot" works as i am new to GUI and QT coding. 
I think this is a official description:  http://doc.qt.io/qt-5/designer-using-a-ui-file.html#widgets-and-dialogs-with-auto-connect

I used it, because all other Slot functions in this widget use the same functionality, and i wanted to keep it consistent in this file.

Maybe adding a comment to this file with a short description and a link to the auto connect documentation is better, than just changing only this function.